eslint.config.mjs

```diff
@@ -140,6 +141,7 @@ export const configs = {
       },
     },
   ],
+  yaml: [...eslintPluginYml.configs['flat/recommended'], ...jsonSchema.configs['flat/recommended']],
```

Comment:
Also add `...eslintPluginYml.configs['flat/prettier']` to turn off rules that might interfere with prettier